tt

```diff
@@ -57,6 +55,10 @@ main() {
 		esac
 	fi
 
+	TT_PROJECTS=${TT_PROJECTS:-~/tmp}
+	TT_GIT_BASE=${TT_GIT_BASE:-git@github.com:meet-tarantino}
+	apply_workspace_vars # this may override the above defaults
```

Comment:
Based on what's in master and https://github.com/meet-tarantino/tarantino/issues/1, our intent was to allow workspaces to contain a `plugins` directory. We would then `.` source the scripts in there so we could inherit other `tt_...` functions. We could leverage that for configuration too?